scripts/script.js

```diff
@@ -0,0 +1,34 @@
+import { translateToMorse, translateToEnglish } from './translator.js';
+
+// DOM Elements
+const userInput = document.getElementById('user-input');
+const translationInput = document.getElementById('translation');
+const translateBtn = document.getElementById('translate-btn');
+const errorDisplay = document.getElementById('error');
+
+translateBtn.addEventListener('click', () => {
+  try {
+    // Clear error display and translation input
+    errorDisplay.textContent = '';
+    translationInput.value = '';
+
+    // Get value of the user input field
+    const userText = userInput.value.trim();
+
+    if (!userText) {
+      // If no input is provided
+      throw new Error('Please enter text to translate.');
+    }
+
+    // if morse code
+    if (/^[\.\-\/\s]+$/.test(userText)) {
+      translationInput.value = translateToEnglish(userText);
+    } else {
+      // else English text
+      translationInput.value = translateToMorse(userText);
+    }
```

Comment:
This could be contained in a function and tested using mocks. A little tricky but kinda cool to ensure that the right function is being called. I'll try and explain in our session